rootfs/etc/profile.d/motd.sh

```
@@ -2,8 +1,11 @@
+if [[ $SHLVL -eq 1 ]] && ! [[ $GEODESIC_MOTD_ENABLED == "false" ]]; then
 	if [ -f "/etc/motd" ]; then
-		cat "/etc/motd"
+		source "/etc/motd.sh"
```

Shellcheck (0.10.0)

[info] Not following: /etc/motd.sh was not specified as input (see shellcheck -x).

(SC1091)